backend/src/main/java/gov/cdc/usds/simplereport/api/queue/ApiTestOrderDataResolver.java

```diff
@@ -100,13 +88,12 @@ public CompletableFuture<LocalDate> symptomOnset(
   }
 
   @SchemaMapping(typeName = "TestOrder", field = "deviceType")
-  public CompletableFuture<DeviceType> deviceType(
-      ApiTestOrder apiTestOrder, DataLoader<UUID, DeviceType> loader) {
-    return loader.load(apiTestOrder.getWrapped().getDeviceType().getInternalId());
+  public DeviceType deviceType(ApiTestOrder apiTestOrder) {
+    return apiTestOrder.getWrapped().getDeviceType();
   }
 
   @SchemaMapping(typeName = "TestOrder", field = "results")
-  public Set<Result> getResults(ApiTestOrder apiTestOrder) {
+  public Set<Result> results(ApiTestOrder apiTestOrder) {
```

Comment:
non-blocking comment: funny, it looks like this was still working anyway?

Reply:
Yeah it was working because it has the `@SchemaMapping,` which bypass the naming requirement, but I just changed it to keep it consistent with the rest of the file.

Reply:
The naming requirements are only `query/mutations` and their `arguments`, not `@SchemaMapping`. You can think of them as Input and Output mapping and the naming requirement only is on the input side.